examples/tensorflow/classification/configs/quantization/inception_v3_imagenet_int8.json

```diff
@@ -24,8 +24,14 @@
         "algorithm": "quantization",
         "initializer": {
             "batchnorm_adaptation": {
-                "num_bn_adaptation_samples": 2048,
+                "num_bn_adaptation_samples": 2048
             }
+        },
+        "weights": {
+            "per_channel": false
```

Comment:
Was it really per-tensor by default before?

Reply:
See the default config:
https://github.com/openvinotoolkit/nncf/pull/836/files#diff-0e36c84e86831a6f19916947f5218473fa59e6654eed3014e5f9c9cd8ac02170L261-L264